src/types.rs

```diff
@@ -555,3 +555,25 @@ impl StructuralType {
         simplicity::types::Type::from(self.0.clone())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn display_type() {
+        let unit = ResolvedType::unit();
+        assert_eq!("()", &unit.to_string());
+        let singleton = ResolvedType::tuple([ResolvedType::from(UIntType::U1)]);
+        assert_eq!("(u1,)", &singleton.to_string());
+        let tuple = ResolvedType::tuple([
+            ResolvedType::from(UIntType::U1),
+            ResolvedType::from(UIntType::U8),
+        ]);
+        assert_eq!("(u1, u8)", &tuple.to_string());
```

Comment:
In 7b7bac7afd0c63871f8bf4c13b2866894015995a:

Should have a test with tuples with more than 2 items.

Reply:
I added a test for triple types and values. Hopefully this clarifies that the Display code is correct.